notebooks/s3d-mil-nce-text-to-video-retrieval/s3d-mil-nce-text-to-video-retrieval.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import tensorflow.compat.v2 as tf
</pre><p>above you install tf_keras package. it is used for switching keras inside tf to v2 instead of default v3 in tf 2.16 and higher. it is also required to use variable</p><p>&nbsp;"os.environ[\"TF_USE_LEGACY_KERAS\"] = \"1\"\n", for that (more details on this page <a href="https://keras.io/getting_started" target="_blank">https://keras.io/getting_started/</a>) before any tensorflow imports.</p><p> </p><p>Could you please check notebook work with the latest tf and clarify versions requirements</p><p> </p><p>Also for avoiding issues with global caching, we prefer redirect hub cache into notebook dir (see for details <a href="https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/tensorflow-hub/tensorflow-hub.ipynb" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/tensorflow-hub/tensorflow-hub.ipynb</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>